README.md

```diff
@@ -17,11 +17,10 @@ This repo includes the following components:
 
 ## Project Setup
 
-After you've succesfully cloned OpsCenter reprository, please run:
+Clone OpsCenter repository by running:
 
 ```
-git submodule init
-git submodule update
+git clone <URL of the repository> --recurse-submodules
```

Comment:
I suggest you include the actual url of the opscenter github repo.

Also not sure but thought flag needed to come before url.

Reply:
I tested before I posted PR, flag does not have to be before URL.  Will change it though, it reads nicer.

```
vickymarkman@Vickys-MBP-2 Downloads % git clone git@github.com:vqmarkman/OpsCenter.git --recurse-submodules 
Cloning into 'OpsCenter'...
remote: Enumerating objects: 248, done.
```

Do you suggest we write something like this for the URL?
`git@github.com:<git user name>/OpsCenter.git `

Reply:
I'd just use https://github.com/sundeck-io/OpsCenter.git

If someone forks, they'll be able to add their own remote as a second remote.

Reply:
also found flaky test, looking into it.